### PR TITLE
✨DCS-1336: Add additional hint for TRANS response from BaSM

### PR DESCRIPTION
**Pull request by connormaglynn**

### CODE REVIEW

File: server/routes/sexController.test.ts [hunk 1]
```diff
@@ -65,10 +65,32 @@ describe('/sex', () => {
...
+        lastName: 'smith',
+      } as Movement)
+      return request(app)
+        .get('/prisoners/12345-67890/sex')
+        .expect(200)
+        .expect('Content-Type', 'text/html; charset=utf-8')
+        .expect(res => {
+          const $ = cheerio.load(res.text)
+          expect($('.govuk-inset-text').text()).toContain(
+            'john smith was identified as transgender on their Person Escort Record. Their registered sex at birth is required to confirm their arrival into this establishment.'
```
Comment (GurnankCheema) [inline server/routes/sexController.test.ts:90]: Should name be capitalised i.e. cap J and S ?

**Review by GurnankCheema — APPROVED** [review]
LGTM

File: server/views/pages/sex.njk [hunk 2]
```diff
@@ -17,6 +19,12 @@
             <span class="govuk-caption-xl"
                   data-qa="prisoner-name">Confirm arrival of {{ data.firstName }} {{ data.lastName }}</span>
 
+            {% if data.gender === 'TRANS' %}
```
Comment (andrewrlee) [inline server/views/pages/sex.njk:22]: is the `if` needed? this page will only show for trans?

Comment (connormaglynn) [reply server/views/pages/sex.njk:22]: Suppose if we get a null value or another gender becomes available later down the line protects us against that confusion?

**Pull request merged**